apps/loan/src/lib/apiCrvusd.ts

```diff
@@ -479,7 +479,7 @@ const loanCreate = {
         : null,
     ])
 
-    const maxRecvs = fulfilledValue(maxRecvsResults) ?? null
+    const maxRecvs: any = fulfilledValue(maxRecvsResults) ?? null
```

Comment:
Try to avoid using `any`, better use `as type` or, if you really don't know the type, use `as unknown`.

Reply:
This is old code I was just fixing the build error. The real type is actually pretty convoluted. I'll fix it though.